src/main/resources/META-INF/rewrite/spring-security-63.yml

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2024 the original author or authors.
+# <p>
+# Licensed under the Moderne Source Available License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://docs.moderne.io/licensing/moderne-source-available-license
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.spring.security6.UpgradeSpringSecurity_6_3
+displayName: Migrate to Spring Security 6.3
+description: >-
+  Migrate applications to the latest Spring Security 6.3 release. This recipe will modify an
+  application's build files, make changes to deprecated/preferred APIs, and migrate configuration settings that have
+  changes between versions.
+tags:
+  - spring
+  - security
+recipeList:
+  - org.openrewrite.java.spring.security6.UpgradeSpringSecurity_6_2
+  - org.openrewrite.java.dependencies.UpgradeDependencyVersion:
+      groupId: org.springframework.security
+      artifactId: "*"
+      newVersion: 6.3.x
+      overrideManagedVersion: false
+  - org.openrewrite.java.dependencies.UpgradeDependencyVersion:
+      groupId: org.springframework.security
+      artifactId: spring-security-oauth2-authorization-server
+      newVersion: 1.3.x
+      overrideManagedVersion: false
```

Comment:
I'm wondering about the ordering here and above; I think we validate the versions we migrate to actually exist so it should not be an issue that `org.springframework.security:*:6.3.x` also matches `spring-security-oauth2-authorization-server`, but perhaps it can't hurt to be sure. Did you check that part?

Reply:
Did not verify that, let me do that!